source/MaterialXView/ShaderValidators/Glsl/GlslValidator.cpp

```diff
@@ -380,14 +380,14 @@ void GlslValidator::bindFixedFunctionViewInformation()
 {
     // Bind projection
     glMatrixMode(GL_PROJECTION);
-    Matrix4x4& projection = _viewHandler->projectionMatrix();
-    glLoadMatrixf(&projection[0]);
+    Matrix44& projection = _viewHandler->projectionMatrix();
+    glLoadMatrixf(&(projection[0][0]));
```

Comment:
This was the reason for the contiguous memory query. std:array is supposed to be contiguous.

Reply:
This should be safe, and it's probably worth adding documentation to this effect in a future update to the vector and matrix classes.